### PR TITLE
fast-track: Add a TCK test to verify that the default ExceptionMapper can be overridden.

### PR DESCRIPTION
**Pull request by WhiteCat22**
The `nomapper` tests seem to test the default `ExceptionMapper`, however, we could use a test to verify that the default `ExceptionMapper` can be overridden.

This test overrides the default `ExceptionMapper` to return status HTTP 512 instead of HTTP 500 and the test client verifies that status HTTP 512 is returned.

### CODE REVIEW

File: jaxrs-tck/src/main/java/ee/jakarta/tck/ws/rs/ee/resource/webappexception/mapper/JAXRSClientIT.java
Comment (jim-krueger) [inline jaxrs-tck/src/main/java/ee/jakarta/tck/ws/rs/ee/resource/webappexception/mapper/JAXRSClientIT.java:35]: Change 
Copyright (c) 2012, 2020, 2021
to
Copyright (c) 2012, 2024

**Review by jamezp — COMMENTED** [review]
Minus the two comments, this looks good to me. Once the changes are made I'll approve it.

File: jaxrs-tck/src/main/java/ee/jakarta/tck/ws/rs/ee/resource/webappexception/mapper/JAXRSClientIT.java
Comment (jim-krueger) [reply jaxrs-tck/src/main/java/ee/jakarta/tck/ws/rs/ee/resource/webappexception/mapper/JAXRSClientIT.java:35]: No, I was incorrect.   The practices for Eclipse projects are different.   https://www.eclipse.org/projects/handbook/#ip-copyright-headers

So 
Copyright (c) 2012, 2020, 2021
should really be changed to
Copyright (c) 2012

**Comment by spericas** [comment]
@WhiteCat22 Could you address the comments pls?

File: jaxrs-tck/src/main/java/ee/jakarta/tck/ws/rs/ee/resource/webappexception/mapper/JAXRSClientIT.java
Comment (jansupol) [reply jaxrs-tck/src/main/java/ee/jakarta/tck/ws/rs/ee/resource/webappexception/mapper/JAXRSClientIT.java:35]: Strange, the document used to allow for a single year, or two years (creation, last modification), and in that manner the maven copyright plugin checks two year-values in the copyright header. I am not a lawyer, but I saw somewhere that a copyright wears off after some ten years, unless updated with a new code (and year).

Comment (jansupol) [reply jaxrs-tck/src/main/java/ee/jakarta/tck/ws/rs/ee/resource/webappexception/mapper/JAXRSClientIT.java:35]: From the same document, FAQ section:
> Do we need to specify a range of years in the copyright statement?

> No. In the past, legal advice was that the year of the initial creation of the content and the year of the last change should be reflected in the copyright header. This is no longer the case. Specify the year that the content was initially created in the copyright statement.

**Comment by WhiteCat22** [comment]
I have some failures locally in the other related tests. So I don't think I can re-use this existing application for this test.

**Comment by WhiteCat22** [comment]
I broke out the test into a new test and application to avoid breaking existing tests.

**Comment by spericas** [comment]
@alwin-joseph Could you verify the new version?

File: jaxrs-tck/src/main/java/ee/jakarta/tck/ws/rs/ee/resource/webappexception/defaultmapper/DefaultExceptionMapperIT.java
Comment (jamezp) [inline jaxrs-tck/src/main/java/ee/jakarta/tck/ws/rs/ee/resource/webappexception/defaultmapper/DefaultExceptionMapperIT.java:63]: Another minor nit, we could just remove this comment.

**Review by jamezp — DISMISSED** [review]
The two minor notes should not hold up merging this PR.

**Comment by alwin-joseph** [comment]
> @alwin-joseph Could you verify the new version?

New test passes with Glassfish8.

**Comment by WhiteCat22** [comment]
Addressed James' comments. Also, I tested this against OpenLiberty and it passes.

**Comment by WhiteCat22** [comment]
@alwin-joseph Thank you!

**Review by jamezp — APPROVED** [review]
Looks good, thank you @WhiteCat22. Passes on WildFly as well.

**Review by jim-krueger — APPROVED** [review]

**Review by alwin-joseph — APPROVED** [review]

**Pull request merged**
